package.json

```diff
@@ -25,6 +25,8 @@
   },
   "homepage": "https://github.com/lob/hapi-rate-limit",
   "dependencies": {
+    "redis": "^2.6.0",
+    "then-redis": "https://registry.npmjs.org/then-redis/-/then-redis-1.3.0.tgz"
```

Comment:
since `then-redis` isn't actually used by this plugin, it would be better if it was a `peerDependency` instead of a direct dependency. you should also list it as a dev dep as well so your tests work.